fairscale/nn/pipe/multiprocess_pipeline.py

```diff
@@ -325,7 +319,6 @@ def compute(
         """Runs tasks with synchronization to copy streams."""
 
```

Comment:
If we do a quick return here for any style that is not MultiProcess, can we get rid of the two if statements below?

Reply:
Great catch! Fixed.